packages/SwingSet/CHANGELOG.md

```diff
@@ -3,6 +3,39 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.23.1-dev.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/swingset-vat@0.23.0...@agoric/swingset-vat@0.23.1-dev.0) (2021-11-24)
+
+
+### ⚠ BREAKING CHANGES
+
+* remove newSwap; replace with constantProduct AMM where needed (#4097)
```

Comment:
In the swingset package, removing newswap doesn't seem like a breaking change. I bet it's just some test reorg that ideally would have been a separate commit. I suppose I reviewed this one but didn't consider how squashing would impact release notes